### ISSUE TITLE
In order to get insight on how users share events, send Insight about which app they choose for sharing

### ISSUE THREAD

**Issue by lemonboston** [schedjoules/java-api-client#18]
see https://github.com/schedjoules/android-event-discovery-sdk/pull/38#issue-196051349

**Comment by lemonboston** [comment]
@dmfs Is it a duplicate of https://github.com/schedjoules/android-event-discovery-sdk/issues/39 or we need to do something on the java-api-client side as well for this, that's why we have this ticket?

**Comment by dmfs** [comment]
No, it's not a duplicate. We need to do something here as well.
